test/src/test/java/org/corfudb/runtime/view/StreamViewTest.java

```diff
@@ -29,16 +29,76 @@
  */
 public class StreamViewTest extends AbstractViewTest {
 
-    @Getter
-    final String defaultConfigurationString = getDefaultEndpoint();
-
     public CorfuRuntime r;
 
     @Before
     public void setRuntime() throws Exception {
         r = getDefaultRuntime().connect();
     }
 
+    @Test
+    public void nonCachableStream() {
```

Comment:
Please add comments whereever possible. It is easy to pick up context when we need to in the future.

Reply:
Agree, let's add some description to this test.